keadm/cmd/keadm/app/cmd/util/edgecoreinstaller.go

```diff
@@ -107,8 +107,8 @@ func (ku *KubeEdgeInstTool) createEdgeConfigFiles() error {
 		}
 
 		if strings.HasPrefix(ku.ToolVersion, "1.2") {
-			edgeCoreConfig.Modules.EdgeHub.TLSPrivateKeyFile = strings.Join([]string{KubeEdgeCloudDefaultCertPath, "edge.key"}, "")
-			edgeCoreConfig.Modules.EdgeHub.TLSCertFile = strings.Join([]string{KubeEdgeCloudDefaultCertPath, "edge.crt"}, "")
+			edgeCoreConfig.Modules.EdgeHub.TLSPrivateKeyFile = strings.Join([]string{KubeEdgeCloudDefaultCertPath, "server.key"}, "")
+			edgeCoreConfig.Modules.EdgeHub.TLSCertFile = strings.Join([]string{KubeEdgeCloudDefaultCertPath, "server.crt"}, "")
```

Comment:
In KubeEdge 1.2, the certificate names in cloudcore and edgecore are edge.crt not server.crt?

Reply:
Yes, they are all edge.* https://github.com/kubeedge/kubeedge/blob/v1.2.1/common/constants/default.go#L12